client/post-editor/editor-drawer/featured-image.jsx

```diff
@@ -41,11 +42,13 @@ class EditorDrawerFeaturedImage extends Component {
 					empty={ ! site || ! post || ! getFeaturedImageId( post ) }
 					onClick={ this.startSelecting }
 					onRemove={ this.removeImage }>
-					<FeaturedImage
-						selecting={ this.state.isSelecting }
-						onImageSelected={ this.endSelecting }
-						site={ site }
-						post={ post } />
+					<MediaListData siteId={ this.props.site.ID }>
```

Comment:
@youknowriad or @iseulde is it safe to use `QueryMedia`? Or is this more appropriate?